gradle/validation/forbidden-apis.gradle

```diff
@@ -49,11 +49,6 @@ allprojects { prj ->
           logger.debug("Signature file omitted (does not exist): ${sig}")
         }
       }
-      
-      // commons-io is special: forbiddenapis has a versioned bundledSignature.
-      bundledSignatures += resolvedMods
-        .findAll { id -> id.group == 'commons-io' && id.name == 'commons-io' }
-        .collect { id -> "${id.name}-unsafe-${id.version}" as String }
```

Comment:
I got this error from forbidden-apis:
```
Bundled signatures resource not found: commons-io-unsafe-2.12.0
```

@uschindler Removing these lines in the build seems ti fix the warning, but I'm not sure if it is the right cure?

Reply:
I see that forbidden-apis probably needs a new file like https://github.com/policeman-tools/forbidden-apis/blob/main/src/main/resources/de/thetaphi/forbiddenapis/signatures/commons-io-unsafe-2.11.0.txt - for 2.12. But if there are no new unsafe APIs in 2.12 we could perhaps use `commons-io-unsafe-2.11.0` for now?

Reply:
Ok, I now understand how this works. I added a hack to use `commons-io-unsafe-2.11.0` in the build for now.

Reply:
Hi, the new version is to new. Can you open an issue or or for forbiddenapis?

As workaround the easiest would be to use the 2.11 signatures file.

I do not see the workaround in this PR... Where is it?

Reply:
Hmm, some missing commits were pushed now. It's just a hacky hardcoding of 2.11.0. But with a new version of forbidden-apis with https://github.com/policeman-tools/forbidden-apis/pull/232 we can remove the hack again.